### PR TITLE
bakerydetails

### PR DESCRIPTION
**Pull request by Parul76**

### CODE REVIEW

File: mrscake/src/components/BakeryDetail.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import { USERS_API_URL } from '../constants/user_api_url.js';
+import './Login.css';
+
+class BakeryDetail extends Component {
+  state = 
+  {
+    
+    mobilePhone: "", 
+    address: "",
+    crediCardNumber: "",
```
Comment (Alina-dev-front) [inline mrscake/src/components/BakeryDetail.jsx]: credit card number is for customer

File: Mrs_Cake/Services/UserService.cs [hunk 2]
```diff
@@ -29,10 +29,14 @@ public User Create(User user)
             return user;
         }
 
-        public void Update(string id, User user) =>
-
+        public void Update(string id, User user)
```
Comment (Alina-dev-front) [inline Mrs_Cake/Services/UserService.cs:32]: here you'd better make if statement which checks if it is a bakery or customer to change different properties

Comment (Parul76) [reply Mrs_Cake/Services/UserService.cs:32]: but we have bakery table so we can change there

File: mrscake/src/components/BakeryDetail.jsx (hunk 1 above)
Comment (Parul76) [reply mrscake/src/components/BakeryDetail.jsx]: removed credit card number

**Review by Alina-dev-front — APPROVED** [review]

**Pull request merged**